config/custom-environment-variables.json

```diff
@@ -14,7 +14,6 @@
   "githubBotPW": "GITHUB_BOT_PASSWORD",
   "fastlyDochubMap": "FASTLY_DOCHUB_MAP",
   "entitlementCollection": "USER_ENTITLEMENT_COL_NAME",
-  "reposBranchesCollection": "REPOS_BRANCHES_COL_NAME",
```

Comment:
Unused almost-duplicate...? There were three of these. Removed here and below. 
`reposBranches` vs `repoBranches` is how to spot the duplicate.